src/Components/Components/src/Routing/OnNavigateArgs.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNetCore.Components.Routing
+{
+    public class OnNavigateArgs
+    {
```

Comment:
I agree with @SteveSandersonMS here WRT the naming. `NavigationContext` sounds better IMO.

Reply:
Yeah, I have no strong preference either way. I can change to `NavigationContext`.